.github/workflows/continuous-integration-workflow.yml

```diff
@@ -26,14 +26,14 @@ jobs:
 
     - name: Build
       run: |
-        cmake -Bbuild -H. -DPRECOMPILED_DEPENDENCIES_DIR=/install -DWITH_VNC=1
```

Comment:
Is it necessary to rename the `build` folder? What does it fix?

Reply:
Unfortunately, `setuptools` also builds the Python package in a directory named `build`. It's a mess if you do `pip install .` in speculos folder...

Reply:
I'd prefer to keep speculos build in the `build` directory and find a workaround for `setuptools` (eg. `--dist-dir` or something like this?)